### DISCUSSION TITLE
Assets path changed after deployment

### DISCUSSION THREAD

**Discussion by dijkr** [statamic/cms#8584] [General]
After deploying my first Laravel-app onto a new server, I noticed Statamic changed the path for its assets. Why doesn't it use the path `/storage/app/public/`? The symbolic links were added with `php artisan storage:link`. I did that after reading: [Laravel - Filesystem](https://laravel.com/docs/10.x/filesystem).

Development server:
`<img src="/images/banners/banner_food.jpeg">`

Deployment server:
`<img src="http://localhost/storage/banner_food.jpeg"/>`

My view just outputs the path that Statamic generated. Is there a reason why this behaviour changed after deployment. Might it have to do with .gitignored files? As far as I know, I did everything default, so I didn't put files into .gitignored by myself.  
`<img src="{{ banner_pm }}"/>`

**Comment by jasonvarga** [comment]
Is `banner_pm` an assets fieldtype? It uses whatever URLs are configured for the filesystem.

https://statamic.dev/assets#containers

**Reply by dijkr** [reply]
**Update 1:** with `npm run dev & php artisan serve` there is no problem. With `npm run build`. Could this has to do with `npm run build` when not setting `STATAMIC_LICENSE_KEY`? It's a study project and I'm writing a deployment instruction.
**Update 2:** I maganed to fix this to 1) delete non super admin user 2) set pro to false. After this, the images load.

Thanks for you reply. Not sure if that has to do with it, but after the latest Statamic update, the path on de development & deployment server are equal:
 `<img src="http://localhost/storage/banner_food.jpeg"/>`

However the image does not load on the deployment server, but is available in both assets and the collection. I had to re-upload the asset after deployment, because .gitignore. 

The filesystem remains:
```
        'public' => [
            'driver' => 'local',
            'root' => storage_path('app/public'),
            'url' => env('APP_URL').'/storage',
            'visibility' => 'public',
            'throw' => false,
        ],
```

Update, I noticed this at the console:
`GET http://localhost/storage/banner_ananas.jpg net::ERR_CONNECTION_REFUSED`

Both .htaccess and httpd.conf seems equal too.

**Comment by dijkr** [comment]
I managed to fix this to 1) delete non super admin users 2) set pro to false. After this, the images load.
